internal/geoipupdate/geoip_updater_test.go

```
@@ -263,6 +263,53 @@ func TestNewUpdaterDoesNotMutateDefaultTransport(t *testing.T) {
 	)
 }
 
+func TestDoesNotRetryProxyConnectForbidden(t *testing.T) {
+	tempDir := t.TempDir()
+
+	proxyAttempts := 0
+	proxyServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, _ *http.Request) {
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(golines)